data_steward/cdr_cleaner/manual_cleaning_rules/survey_version_info.py

```diff
@@ -69,9 +69,10 @@ def __init__(self,
                  project_id,
                  dataset_id,
                  sandbox_dataset_id,
-                 cope_lookup_dataset_id,
-                 cope_table_name,
-                 table_namer=None):
+                 table_namer=None,
+                 cope_lookup_dataset_id=None,
+                 cope_table_name=None,
```

Comment:
I'm curious what the purpose of making `cope_lookup_dataset_id` and `cope_table_name` keyword parameters is?